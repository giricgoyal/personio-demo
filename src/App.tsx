import * as React from 'react'
import Navbar from './components/navbar'
import Router from './router'

export default function App(): React.ReactElement {
    return (
        <div className="app">
            <Navbar />
            <div className="app__page">
                <Router />
            </div>
        </div>
    )
}
