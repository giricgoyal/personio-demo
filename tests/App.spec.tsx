import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import App from 'src/App'

describe('App', () => {
    let container
    const renderer = new ShallowRenderer()

    beforeEach(() => {
        container = renderer.render(<App />)
    })

    test('should render and match snapshot', () => {
        expect(container).toMatchSnapshot()
    })
})
