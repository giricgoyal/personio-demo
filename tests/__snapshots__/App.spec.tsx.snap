// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render and match snapshot 1`] = `
<div
  className="app"
>
  <Navbar />
  <div
    className="app__page"
  >
    <Router />
  </div>
</div>
`;
